jupyterhub/values.yaml

```diff
@@ -48,8 +48,7 @@ hub:
   image:
     name: jupyterhub/k8s-hub
     tag: 'set-by-chartpress'
-    # pullSecrets:
-    #   - secretName
+    pullSecrets: []
```

Comment:
```suggestion
    pullSecrets:
```

Reply:
config c.KubeSpawner.image_pull_secrets = Unicode(None)

The kubernetes secret to use for pulling images from private repository.

Set this to the name of a Kubernetes secret containing the docker configuration required to pull the image.

See the Kubernetes documentation for more information on when and why this might need to be set, and what it should be set to.

We couldn't set pullSecret config to array type because Kubespawner takes that configs as Unicode string so I have set it to string type on my production deployment with existing image pull secrets :)